src/sentry/static/sentry/app/components/events/interfaces/spans/spanBar.tsx

```diff
@@ -576,20 +579,31 @@ class SpanBar extends React.Component<SpanBarProps, SpanBarState> {
     }
 
     return (
-      <WarningTextWrapper>
-        <Tooltip title={warningText}>
-          <span style={{marginLeft: '8px', lineHeight: 0, height: '15px'}}>
-            <WarningIcon />
-          </span>
-        </Tooltip>
-      </WarningTextWrapper>
+      <Tooltip title={warningText}>
+        <WarningIcon src="icon-circle-exclamation" />
+      </Tooltip>
     );
   };
 
+  getDurationDisplay = ({width, left}) => {
```

Comment:
nit: We could pull this out of the class, since it doesn't need access to the class instance.

Can we also type the argument? You may need to refactor the type for `getBounds` so it is not inline.